bdsf/functions.py

```diff
@@ -2268,3 +2269,54 @@ def centered(arr, newshape):
     endind = startind + newshape
     myslice = [slice(startind[k], endind[k]) for k in range(len(endind))]
     return arr[tuple(myslice)]
+
+
+def set_up_output_paths(opts):
```

Comment:
The name 'base' is a bit confusing here. Could you add in the docstring what `parentname` and `basename` mean? I think a more descriptive name for `basename` would be `output_basedir`? (Intuitively, basename means the basename of the image directory for me.)

Reply:
Yes, good point. I made the suggested changes.